index.html

```diff
@@ -107,40 +107,58 @@ <h3>Multi-Post <br> Stories</h3>
+                <li><i class="fa-solid fa-angle-down"></i></li>
             </ul>
-            <button class="resume-mobile-btn" type="submit">Get my resume</button>
-            <ul class="lang-area">
-                <ul class="stacknav">
-                    <li><strong>Languages</strong></li>
-                    <li><i class="fa-solid fa-angle-down"></i></li>
-                </ul>
-                <ul class="stacks">
-                    <li><button class="html" type="submit"><a href="#">HTML</a></button></li>
-                    <li><button class="css" type="submit"><a href="#">CSS</a></button></li>
-                    <li><button class="js" type="submit"><a href="#">JavaScript</a></button></li>
+            <ul class="stacks">
+                <li><button class="html" type="submit"><a href="#">HTML</a></button></li>
+                <li><button class="css" type="submit"><a href="#">CSS</a></button></li>
+                <li><button class="js" type="submit"><a href="#">JavaScript</a></button></li>
+            </ul>
+            <ul>
+                <ul class="frameworks">
+                    <li><strong>Frameworks</strong></li>
+                    <li><i class="fa-solid fa-angle-right"></i></li>
                 </ul>
-                <ul>
-                    <ul class="frameworks">
-                        <li><strong>Frameworks</strong></li>
-                        <li><i class="fa-solid fa-angle-right"></i></li>
-                    </ul>
-                    <ul class="skills">
-                        <li><strong>Skills</strong></li>
-                        <li><i class="fa-solid fa-angle-right"></i></li>
-                    </ul>
+                <ul class="skills">
+                    <li><strong>Skills</strong></li>
+                    <li><i class="fa-solid fa-angle-right"></i></li>
                 </ul>
             </ul>
-        </section>
+        </ul>
+    </section>
+        <!-- contact me  -->
+    <section class="contact">
+        <div class="contact-bg-img">
+            <h6>Contact me</h6>
+            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it!</p>
+            <form action="">
+                <label for="name">Name:</label>
+                <input type="name" id="name" name="user_name" placeholder="Lorem Ipsum" maxlength="30" required>
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="user_name" placeholder="loremipsum@gmail.com" required>
+                <label for="msg">Message:</label>
+                <textarea name="user_name" id="msg" cols="15" rows="10" maxlength="500" required>Write your message here...</textarea>
+                <button class="contact-mobile-btn" type="submit">Get in touch</button>
+                <img src="/images/home-btn.png" alt="home-button" class="home-btn">
```

Comment:
- [ ] Since this image is just a decoration it will be nice to use the CSS property `background-image` to implement it for semantic reasons.